src/pages/Browse/index.spec.tsx

```diff
@@ -83,6 +84,12 @@ describe(__filename, () => {
       _viewVersionFile,
     };
 
+    // Sometimes we need to mock `dispatch` to prevent it from dispatching
+    // actions which would result in an unhandled promise rejection.
```

Comment:
Although the unhandled rejection is obvious, the result is more that it's going to call a fetch mock that will return a falsey response, leading to the rejection if we try to read data from it